### PR TITLE
Updated Temporary Edit Profile Page and Created Searchable Drop Down List

### PR DESCRIPTION
**Pull request by sammyli0106**
1. Updated the Edit Profile Page to be temporary (navigation need to be revised @zhao19991111 )
2. Added the searchable drop down list for the edit profile page

### CODE REVIEW

File: frontend/src/components/DropDown/index.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  { label: 'Lin 170', value: 'Lin 170' },
+  { label: 'CS 35L', value: 'CS 35L' },
+  { label: 'Math 170A', value: 'Math 170A' },
+  { label: 'Eng 100', value: 'Eng 100' },
+  { label: 'Chem 100', value: 'Chem 100' },
+];
+
+const AppDropDown = ({children, style}) => {
+    return (
+        <Select style={{ ...styles.button, ...style }}
```
Comment (zhao19991111) [inline frontend/src/components/DropDown/index.jsx:19]: It looks like the styling of react-select is based on a "styles" attribute with its own functions

File: frontend/src/components/DropDown/styles.jsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export default {
```
Comment (zhao19991111) [inline frontend/src/components/DropDown/styles.jsx:1]: The styles.jsx might not be useful

**Review by zhao19991111 — CHANGES_REQUESTED** [review]

File: frontend/src/components/DropDown/index.jsx (hunk 1 above)
Comment (sammyli0106) [reply frontend/src/components/DropDown/index.jsx:19]: Ok, but the drop down box right now looks squash in the middle. I will try to look into the attached style attribute.

File: frontend/src/components/DropDown/styles.jsx (hunk 2 above)
Comment (sammyli0106) [reply frontend/src/components/DropDown/styles.jsx:1]: ok, i will remove that

**Comment by sammyli0106** [comment]
Fixed the size of the drop down list and combined into the reset pwd update pull request.

**Pull request closed without merging**